diff --git a/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBAppModule.m b/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBAppModule.m
index f679daa..822e31f 100644
--- a/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBAppModule.m
+++ b/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBAppModule.m
@@ -16,6 +16,7 @@
  */
 
 #import <Firebase/Firebase.h>
+#import <React/RCTBridgeModule.h>
 #import <React/RCTUtils.h>
 
 #import "RNFBAppModule.h"
diff --git a/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBUtilsModule.m b/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBUtilsModule.m
index 7d28331..d5cd1dd 100644
--- a/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBUtilsModule.m
+++ b/node_modules/@react-native-firebase/app/ios/RNFBApp/RNFBUtilsModule.m
@@ -16,6 +16,7 @@
  */
 
 #import <React/RCTUtils.h>
+#import <React/RCTBridgeModule.h>
 
 #import "RNFBApp/RNFBSharedUtils.h"
 #import "RNFBUtilsModule.h"
